hi-ml-azure/src/health_azure/himl.py

```diff
@@ -381,10 +472,44 @@ def _str_to_path(s: Optional[PathOrString]) -> Optional[Path]:
     return s
 
 
+def create_v2_inputs(ml_client: MLClient, input_datasets: List[DatasetConfig]) -> Dict[str, Input]:
+    inputs: Dict[str, Input] = {}
+    for input_dataset in input_datasets:
+        version = input_dataset.version or 1
+        data_asset: Data = ml_client.data.get(input_dataset.name, version=str(version))
+        data_path = data_asset.id or ""
+        # Note that there are alternative formats that the input path can take, such as:
+        # v1_datastore_path = f"azureml://datastores/{input_dataset.datastore}/paths/<path_to_dataset>"
+        # v2_dataset_path = f"azureml:{input_dataset.name}:1"
+
+        inputs[INPUT_DATASETS_ARG_NAME] = Input(  # type: ignore
+            type=AssetTypes.URI_FOLDER,  # type: ignore
+            path=data_path,
+            mode=InputOutputModes.MOUNT,
+        )
+    return inputs
+
+
+def create_v2_outputs(output_datasets: List[DatasetConfig]) -> Dict[str, Output]:
+    outputs = {}
+    for output_dataset in output_datasets:
+        v1_datastore_path = f"azureml://datastores/{output_dataset.datastore}/paths/{output_dataset.name}"
+        # Note that there are alternative formats that the input path can take, such as:
```

Comment:
Here's the link: https://learn.microsoft.com/en-us/azure/machine-learning/how-to-read-write-data-v2?tabs=cli#supported-paths 
I'm hesistant to include it in the codebase since then we will have to maintain it (check the link still exists). Is there another way I can improve the description?